strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout/strongbox-storage-maven-layout-provider/src/main/java/org/carlspring/strongbox/repository/MavenRepositoryFeatures.java

```diff
@@ -68,10 +71,32 @@ public void init()
 
     public void removeTimestampedSnapshots(String storageId,
                                            String repositoryId,
-                                           String artifactPath,
+                                           String artificatPath,
```

Comment:
There is a typo here.
```suggestion
                                           String artifactPath,
```